src/Bridge/Symfony/Resources/translations/SonataFormBundle.ru.xliff

```diff
@@ -14,6 +14,10 @@
                 <source>label_type_no</source>
                 <target>нет</target>
             </trans-unit>
+            <trans-unit id="sonata_form_template_box_file_found_in">
+                <source>sonata_form_template_box_file_found_in</source>
+                <target>Этот файл можно найти в</target>
+            </trans-unit>
```

Comment:
This seems weird to add a translation only for russian.

Reply:
I copy it from other branch.

Reply:
Ok, I think we should remove this on the 1.x branch then